### PR TITLE
Attach bubble paws into the dual-Jaco setup

### PR DESCRIPTION
**Pull request by aykut-tri**
Couldn't get to attach the paw as a separate link easily, we can look into that later if necessary.

Also adding the redundant paw sdf as a resource.

### CODE REVIEW

File: models/jaco_bubble_picoflexx_paw.obj [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/aykut/.cache/bazel_external_data/sha512/86/c2/86c29a820005579426df388f954a4cc202e8f322929e6541a4926ccf6be6b6956c08218aeb9df61c7f4dd4f7c97528fe714100cf0796f4d156ac61e205cb6fde
```
Comment (JoseBarreiros-TRI) [inline models/jaco_bubble_picoflexx_paw.obj:1]: pls upload the obj file or use the sha pointing to a public repository.

**Comment by JoseBarreiros-TRI** [comment]
@aykut-tri pls upload the obj. I can't test this without it.

File: models/jaco_bubble_picoflexx_paw.sdf [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
```
Comment (JoseBarreiros-TRI) [inline models/jaco_bubble_picoflexx_paw.sdf:1]: nit: pls add color to the plastic part of the paw, and maybe for the bubble too, use white.

**Comment by JoseBarreiros-TRI** [comment]
LGTM after a nit fix.

File: models/jaco_bubble_picoflexx_paw.obj (hunk 1 above)
Comment (aykut-tri) [reply models/jaco_bubble_picoflexx_paw.obj]: done, thanks

File: models/jaco_bubble_picoflexx_paw.sdf (hunk 2 above)
Comment (aykut-tri) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: I don't know how to add color to mesh (i.e., `.obj` in this case), any pointers?

Comment (JoseBarreiros-TRI) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: something like this:
      <geometry>
        <mesh filename="package://drake/manipulation/models/jaco_description/meshes/arm_half_1.obj"/>
      </geometry>
      <material>
        <color rgba="0.1788232 0.1788232 0.1788232 0.8"/>
      </material>
    </visual>

Comment (JoseBarreiros-TRI) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: <visual>
      <geometry>
        <mesh filename="package://drake/manipulation/models/jaco_description/meshes/arm_half_1.obj"/>
      </geometry>
      <material>
        <color rgba="0.1788232 0.1788232 0.1788232 0.8"/>
      </material>
    </visual>

Comment (aykut-tri) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: The whole mesh is already colored in the sdf (by a whitish color), see the diffuse attribute below:
```
<visual name="base_visual">
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <scale>1 1 1</scale>
            <uri>jaco_bubble_picoflexx_paw.obj</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>0.3 0.3 0.3 1</ambient>
          <diffuse>0.7 0.7 0.7 0.5</diffuse>
          <specular>0.001 0.001 0.001 1</specular>
          <emissive>0 0 0 1</emissive>
        </material>
      </visual>
```

But I don't know how to color individual parts in the mesh, such as the plastic part and the bubble in your request. Though, I can make the whole mesh blackish and the bubble collision sphere white as below:
black_paw

Comment (JoseBarreiros-TRI) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: got it, I thought they have two separate meshes. The black paw looks good anyways.

Comment (aykut-tri) [reply models/jaco_bubble_picoflexx_paw.sdf:1]: Cool, done and merging

**Pull request merged**